<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/qtEditIcon/beautify_letter_background.jpg);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionNEW_FILE"/>
    <addaction name="actionOPEN_FILE_O"/>
    <addaction name="actionSAVE_FILE_S"/>
    <addaction name="separator"/>
    <addaction name="actionQUIT"/>
   </widget>
   <widget class="QMenu" name="menuEDIT">
    <property name="title">
     <string>EDIT</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionSelect"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSetFont_T"/>
    <addaction name="actionSetColor"/>
    <addaction name="actionSetDateTime_D"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEDIT"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSAVE_FILE_S"/>
   <addaction name="actionCopy"/>
   <addaction name="actionSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNEW_FILE">
   <property name="text">
    <string>NEW FILE(&amp;N)</string>
   </property>
  </action>
  <action name="actionOPEN_FILE_O">
   <property name="text">
    <string>OPEN FILE(&amp;O)</string>
   </property>
  </action>
  <action name="actionSAVE_FILE_S">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/qtEditIcon/save.png</normaloff>:/new/prefix1/qtEditIcon/save.png</iconset>
   </property>
   <property name="text">
    <string>SAVE FILE(&amp;S)</string>
   </property>
  </action>
  <action name="actionQUIT">
   <property name="text">
    <string>QUIT(&amp;Q)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo(&amp;Z)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo(&amp;Y)</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/qtEditIcon/cut.png</normaloff>:/new/prefix1/qtEditIcon/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/qtEditIcon/copy.png</normaloff>:/new/prefix1/qtEditIcon/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste(&amp;V)</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll(&amp;A)</string>
   </property>
  </action>
  <action name="actionSetFont_T">
   <property name="text">
    <string>SetFont(&amp;T)</string>
   </property>
  </action>
  <action name="actionSetColor">
   <property name="text">
    <string>SetColor(&amp;C)</string>
   </property>
  </action>
  <action name="actionSetDateTime_D">
   <property name="text">
    <string>SetDateTime(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
